x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_partition_field/categorization_per_partition.tsx

```diff
@@ -32,14 +32,7 @@ export const CategorizationPerPartitionField: FC = () => {
 
   return (
     <Description>
-      <EuiFormRow
-        label={
-          <FormattedMessage
-            id="xpack.ml.newJob.wizard.extraStep.categorizationJob.perPartitionCategorizationLabel"
-            defaultMessage="Enable per-partition categorization"
-          />
-        }
-      >
+      <EuiFormRow>
```

Comment:
>       <CategorizationPerPartitionSwitch />

In https://github.com/elastic/kibana/pull/149086 we recently changed the label on a toggle like this from "Enable..." to "On" and "Off", which is what you see in Advanced Settings too.  Something to maybe consider here too.

Reply:
I took note of it and will address it in a separate PR.